src/rules/curlyRule.ts

```diff
@@ -19,6 +19,12 @@ import * as ts from "typescript";
 
 import * as Lint from "../lint";
 
+const OPTION_ALL = "all";
```

Comment:
rename to "always" to be consistent with other rules (I realize this is different from eslint)